src/main/webapp/filterStyle.css

```diff
@@ -1,34 +1,33 @@
-/* TODO: Move some of these in to common CSS file once next round of PRs are merged in. */
-
-.site-background {
-  background: #05b091;
-  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
-}
 
 .full-page {
   display: flex;
   flex-flow: column wrap;
 }
 
 .header {
-  background: #92b9dd;
-  border: 1px solid #03635d;
+  background: #5d5179;
+  border: 1px solid white;
   border-radius: 15px;
+  color: white;
   box-sizing: border-box;
+  font-family: 'Playfair Display', serif;
   font-size: 60px;
   height: 200px;
   margin: auto;
+  margin-top: 100px;
   padding: 50px;
   text-align: center;
   width: 600px;
 }
 
 .description {
-  background: #69d1dd;
-  border: 1px solid #03635d;
+  background: #92b9dd;
+  border: 1px solid black;
   border-radius: 15px;
   box-sizing: border-box;
+  color: black;
   height: 100px;
+  font-family: 'Raleway', sans-serif;
```

Comment:
Is this font setting necessary given that the common styling also sets fonts?

Reply:
Since the common styling sets fonts for paragraph tags, anchor tags, and headers, and my html is pretty much all divs, I needed to include it in my styling too.